apps/zero-runtime-vite-app/.eslintrc

```diff
@@ -1,5 +1,8 @@
 {
   "env": {
     "jest": true
+  },
+  "rules": {
+    "react/react-in-jsx-scope": "off"
```

Comment:
Why block this? We can import React where it makes sense.

Reply:
We should rather be consistent in the repo.

Reply:
It's a vite app with `jsx` as auto. So it's not needed to import React if you are only using JSX

Reply:
Hi @mnajdova Could you look into the PR again? I've implemented majority of the requested changes.